.github/workflows/deploy.yml

```diff
@@ -175,4 +175,4 @@ jobs:
             dbildungscloud-iam-keycloak \
             --namespace ${{ inputs.namespace }} \
             --kubeconfig $(pwd)/kubeconfig \
-            --set keycloakHostname=${{ needs.create_lowercase_ingress_prefix.outputs.namespace_from_branch }}keycloak.dev.spsh.dbildungsplattform.de
\ No newline at end of file
+            --set keycloakHostname=${{ needs.create_lowercase_ingress_prefix.outputs.namespace_from_branch }}.keycloak.dev.spsh.dbildungsplattform.de
```

Comment:
When I tried that last (admittedly months ago) it led to the certificates not working because they only apply to the first level beneath dev.spsh....

Hyphenation works though.

Also: I need all three backend, frontend, keycloak on all deployments